### PR TITLE
TargetFrameworks article

### PR DESCRIPTION
**Pull request by csharpfritz**
Target Frameworks supported by NuGet official clients defined and documented in response to NuGet/Home#358

### CODE REVIEW

**Comment by csharpfritz** [comment]
Tagging folks for review before merge:
@emgarten @davidfowl @ericstj @yishaigalatzer

File: NuGet.Docs/Create/TargetFrameworks.md [hunk 1]
```diff
@@ -0,0 +1,647 @@
+## Target Frameworks
+
+As the .NET ecosystem has grown, so too has the list of frameworks and dependencies that NuGet supports.   The frameworks that can be included and referenced by a NuGet package tracks closely with the list of available .NET framework versions and flavors that support all of the devices and systems the .NET framework can be used on.
+
+Target frameworks can be referenced from three places:  nuspec manifest, nupkg folder name, and project.json frameworks node.  A framework is typically referenced by a short name, called a "Target Framework Moniker" or TFM.  With the advent of the new Platform Standard, the concept of 'TFM' has been abstracted to 'TxM' since the Platform Standard references multiple frameworks abstractly.
```
Comment (davidfowl) [inline NuGet.Docs/Create/TargetFrameworks.md]: I'd be more specific about this. Explain how they manifest in the nuspec (like on the dependencies element) and in project.json.

Also explain you can have assets that vary by TFM (certain ones).

File: NuGet.Docs/Create/TargetFrameworks.md [hunk 2]
```diff
@@ -0,0 +1,647 @@
...
+    <td>xamarinxboxone</td>
+    <td>Xamarin for XBox One</td>
+    <td><i>dotnet5.5</i><br/>netstandard1.4</td>
+  </tr>
+</table>
+
+Stephen Cleary has a blog post that demonstrates a tool that will mine the list of supported portable class libraries on a workstation and report the features of them: [http://blog.stephencleary.com/2012/05/framework-profiles-in-net.html](http://blog.stephencleary.com/2012/05/framework-profiles-in-net.html)
+
+
+### .NET Platform Standard
```
Comment (davidfowl) [inline NuGet.Docs/Create/TargetFrameworks.md]: Explain that this isn't part of 3.3 the new netstandard tfm will be coming in 3.4

File: NuGet.Docs/Create/TargetFrameworks.md (hunk 1 above)
Comment (csharpfritz) [inline NuGet.Docs/Create/TargetFrameworks.md]: Added and updated content with add links to other docs with more details

File: NuGet.Docs/Create/TargetFrameworks.md (hunk 2 above)
Comment (csharpfritz) [inline NuGet.Docs/Create/TargetFrameworks.md]: Added note about version change coming in 3.4+

File: NuGet.Docs/Create/TargetFrameworks.md [hunk 3]
```diff
@@ -0,0 +1,662 @@
...
+				<li>dnx45</li>
+				<li>dnx451</li>
+				<li>dnx452</li>
+			</ul>
+		</td>
+	</tr>
+	<tr>
+		<td>DNX Core</td>
+		<td>dnxcore</td>
+		<td>Cross-Platform server-side framework support
```
Comment (davidfowl) [inline NuGet.Docs/Create/TargetFrameworks.md]: It's not cross platform. It's core clr based implementations

**Comment by csharpfritz** [comment]
Are we good to merge and publish?

**Pull request merged**